### ISSUE TITLE
RequestRatingView does not open with "Exit"

### ISSUE THREAD

**Issue by itagagaki** [1Remote/1Remote#788]
When the user clicks the X button in the main window, the `RequestRatingView` window will appear if the conditions are met.
However, if the user exits the application with “Exit” from the window or icon menu, it will not appear.
It would be better if the same process was done on each path that exits the application.

**Comment by itagagaki** [comment]
Ah, so the X button simply closes the main window, but does not terminate the process (the other windows remain), and Exit terminates the process. I hesitate if users will understand this difference.

**Comment by VShawn** [comment]
> Ah, so the X button simply closes the main window, but does not terminate the process (the other windows remain), and "Exit" terminates the process. I hesitate if users will understand this difference.

That's right, it's exactly as you said. And most users can understand that clicking X button will minimiz to the system tray.

A better approach is to provide an animation to show the procedure `minimizes to the tray` after clicking the close button, just like in macOS. However, I don't know how to implement this animation in WPF.

**Comment by itagagaki** [comment]
Yeah, it would certainly be easier to understand if it could be done like macOS, but that's not possible to do that with other apps, so it's not an option.

At the moment, the only trigger to bring up a dialog is the Close button, but if you think that is sufficient, that is fine.

**Comment by VShawn** [comment]
Yes, I think one entrypoint is enough, especially I don't want RequestRatingView to bother users too much.
